source/MRMesh/MRGcodeProcessor.cpp

```diff
@@ -14,6 +14,12 @@ constexpr int pointInRotation = 21;
 //////////////////////////////////////////////////////////////////////////
 // GcodeExecutor
 
+GcodeProcessor::GcodeProcessor()
+{
+    rotationAxes_ = { Vector3f::minusX(), Vector3f::minusY(), Vector3f::plusZ() };
+    setRotationOrder( { RotationParameterName::A, RotationParameterName::B, RotationParameterName::C } );
+}
+
```

Comment:
why do in constructor, mb just initialize it in class body?

Reply:
fixed